pkg/config/config_template.yaml

```diff
@@ -1283,6 +1283,14 @@ api_key:
   # max_cpu_percent: 50
 
   ## @param obfuscation - object - optional
+  ## @param DD_APM_OBFUSCATION_CREDIT_CARDS_ENABLED - boolean - optional
+  ## @param DD_APM_OBFUSCATION_CREDIT_CARDS_LUHN - boolean - optional.
+  ## @param DD_APM_OBFUSCATION_MONGODB - object - optional
+  ## @param DD_APM_OBFUSCATION_ELASTICSEARCH - object - optional
+  ## @param DD_APM_OBFUSCATION_REDIS_ENABLED - boolean - optional
+  ## @param DD_APM_OBFUSCATION_MEMCACHED_ENABLED - boolean - optional
+  ## @param DD_APM_OBFUSCATION_HTTP - object - optional
+  ## @param DD_APM_OBFUSCATION_REMOVE_STACK_TRACES - boolean - optional
```

Comment:
I do believe these should be documented separately as well, and have separate subsections under `obfuscation`. Let's also make sure we capture [all the obfuscation options](https://github.com/DataDog/datadog-agent/blob/be63ac38c088c6eb7b11834cf3202dfef95e9e0f/pkg/config/apm.go#L23-L39) that we support.